### PR TITLE
CPR-329 rename person tables

### PR DESCRIPTION
**Pull request by H-Iwanejko**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/person/Person.kt [hunk 1]
```diff
@@ -25,29 +25,13 @@ data class Person(
...
-  val nationalityCode: String? = null,
   val nationalInsuranceNumber: String? = null,
-  val observedEthnicityDescription: String? = null,
-  val selfDefinedEthnicityDescription: String? = null,
-  val homePhone: String? = null,
-  val mobile: String? = null,
-  val workPhone: String? = null,
-  val primaryEmail: String? = null,
-  val contacts: List<PersonContact> = emptyList(),
-  val address: List<PersonAddress> = emptyList(),
```
Comment (H-Iwanejko) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/person/Person.kt:50]: I've removed redundant values here to simplify

**Review by markreesmoj — APPROVED** [review]
👍

**Comment by markreesmoj** [comment]
Might be worth scaling down to 0 instances before starting the production deployment to avoid any problems with the pod running the old code?

**Comment by H-Iwanejko** [comment]
Yeah that is a good point, ill raise a PR

**Pull request merged**